### ISSUE TITLE
Old almanac combat training not firing

### ISSUE THREAD

**Issue by thejko** [rpherbig/dr-scripts#3597]
I have 
```
training_abilities:
  Almanac: 120
```
However the almanac is never used during combat despite being selected. When I use debug the following occurs:

```
[combat-trainer: Selected: Almanac]

> 
You project your voice quietly towards a long-fanged  warcat, freezing it in place with the primal horror of your whispers.

> 
[combat-trainer: Selected: Ambush Stun]
```
It seems to properly select almanac, however nothing happens.

**Comment by Sarvatt** [comment]
will be fixed by https://github.com/rpherbig/dr-scripts/issues/3597

**Comment by rpherbig** [comment]
Fixed in #3596 e7310ce44fa83af6f61b222b671f7141b5630157
